git_app/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# Varangian
+# Copyright(C) 2020 Kevin Postlethwait
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Templates for issues and PRs opened by Varangian git application."""
+
+__name__ = "varangian_git_app"
+__version__ = "0.1.0"
```

Comment:
```suggestion
__version__ = "0.0.0"
```

As this project is not yet released , it should have any version.

Reply:
++ thanks harshad